LoopKit/Service/RemoteDataService.swift

```diff
@@ -118,5 +118,16 @@ public extension RemoteDataService {
     var pumpEventDataLimit: Int? { return nil }
     var settingsDataLimit: Int? { return nil }
 
```

Comment:
Replace the signature with a Result type so I can capture the Error. I'm partial towards throwing APIs in Swift these days but I see Result types more common throughout Loop.

Reply:
When the error path is correctly feeding errors to the caregiver app, the flow of errors here will be obviated as well.